dashboard/index.php

```diff
@@ -19,34 +19,46 @@
 
 $Reflector->LoadXML();
 
+function updateHashFile($HashFile, $newLastSync, $newHash) {
+   // Open the file for writing only, If the file does not exist, it is created.
+   // If it exists, it is neither truncated (as opposed to 'w'), nor the call to this function fails
+   // The file pointer is positioned on the beginning of the file. we'll ftruncate() after writing.
+   $Ressource = @fopen($HashFile, "c");
+
+   if ($Ressource) {
+      // exclusive lock
+      if (flock($Ressource, LOCK_EX)) {
+         @fwrite($Ressource, "<?php\n");
+         @fwrite($Ressource, "\n".'$LastSync = "'.$newLastSync.'";');

```

Comment:
this should be:

`@fwrite($Ressource, "\n".'$LastSync = '.$newLastSync.';');`

because $newLastSync is an integer, not a string. So no quotes are needed and it may break when including the hashfile and subtract the current timestamp from the $LastSync value. I tested this and it doesn't break. But still...

Reply:
thanks for your comments @remcovz 

I am traveling, will address your comments when I get back home tomorrow.